simplic-authentication/Simplic.Authentication/Interface/IAuthenticationService.cs

```diff
@@ -1,18 +1,27 @@
-﻿namespace Simplic.Authentication
+﻿using Simplic.Session;
+
+namespace Simplic.Authentication
 {
     /// <summary>
     /// Authentication service handles logging a user in and holding the current session data
     /// </summary>
     public interface IAuthenticationService
     {
         /// <summary>
-        /// Authenticate a user and create a use session
+        /// Authenticate a user and create a user session
         /// </summary>
         /// <param name="domain">Domain</param>
         /// <param name="userName">User name</param>
         /// <param name="password">Password</param>
         /// <returns>A user session if the user could be logged in, else an exception will be thrown</returns>
-        Simplic.Session.Session Login(string domain, string userName, string password);
+        Session.Session Login(string domain, string userName, string password);
+
+        /// <summary>
+        /// Authenticate a user and create a user session
+        /// </summary>
+        /// <param name="externAccountName">External account name</param>
+        /// <returns>A user session</returns>
+        Session.Session LoginByExternAccount(string externAccountName);        

```

Comment:
Is the account name enough?

Reply:
account typeid was not used at all. so I removed it

Reply:
What do you mean with not used? The external account system is made for this. We should take a look at it together. I think for office 365 this was used.

Reply:
check this line here: https://github.com/simplic/simplic-framework/blob/8c6079fdba6c5888451af0ffbbf033a34fca0fb1/src/Simplic.Framework.UI/Core/User/UserManager.cs#L306